definitions/scenarios/upgrade_to_capsule_6_7.rb

```diff
@@ -0,0 +1,83 @@
+module Scenarios::Capsule_6_7
+  class Abstract < ForemanMaintain::Scenario
+    def self.upgrade_metadata(&block)
+      metadata do
+        tags :upgrade_scenario
+        confine do
+          feature(:capsule) && feature(:capsule).current_minor_version == '6.6'
+        end
+        instance_eval(&block)
+      end
+    end
+
+    def target_version
+      '6.7'
+    end
+  end
+
+  class PreUpgradeCheck < Abstract
+    upgrade_metadata do
+      description 'Checks before upgrading to Capsule 6.7'
+      tags :pre_upgrade_checks
+      run_strategy :fail_slow
+    end
+
+    def compose
+      add_steps(find_checks(:default))
+      add_steps(find_checks(:pre_upgrade))
+      add_step(Checks::RemoteExecution::VerifySettingsFileAlreadyExists.new)
```

Comment:
So it was because of this.  
@upadhyeammit so this check is required for capsule upgrade?